docs/output.md

```diff
@@ -1,45 +1,299 @@
+This subdirectory contains the output of running `run_dbCAN` on the assembly proteins, generating annotations for CAZymes and CAZyme Gene Clusters (CGC).
+
+- **ERZ12345_dbcan_cgc.gff.gz**: This `gff` file is annotated with functional genes for CGCFinder and visualization.
+- **ERZ12345_dbcan_standard_out.tsv.gz**: This `tsv` file lists all identified CGCs and their components.
+- **ERZ12345_dbcan_overview.tsv.gz**: This `tsv` file contains a summary of identified CAZymes.
+- **ERZ12345_dbcan_sub_hmm.tsv.gz**: This file `tsv` file contains the detailed HMMER results.
+- **ERZ12345_dbcan_substrates.tsv.gz**: This `tsv` file contains the dbCAN sub-HMM results, including substrate specificity.
+
+### pathways-and-systems
+
+The `pathways-and-systems` directory contains five subdirectories of results, one for each pathway/system annotation tool used by the pipeline. The results range from KEGG pathways, to Biosynthetic Gene Clusters (BGCs) by `antiSMASH` and `SanntiS`. Just like the functional annotations, most of these annotations are on a per-protein basis.
+
+```bash
+├── qc
+├── cds
+├── taxonomy
+├── functional-annotation
+├── pathways-and-systems
+│   ├── antismash
+│   │   ├── ERZ12345_antismash.gbk.gz
+│   │   ├── ERZ12345_antismash.gff.gz
+│   │   ├── ERZ12345_merged.json
+│   │   └── ERZ12345_antismash_summary.tsv.gz
+│   ├── sanntis
+│   │   └── ERZ12345_sanntis_concatenated.gff.gz
+│   ├── genome-properties
+│   │   ├── ERZ12345_gp.json.gz
+│   │   ├── ERZ12345_gp.tsv.gz
+│   │   └── ERZ12345_gp.txt.gz
+│   ├── kegg-modules
+│   │   ├── ERZ12345_kegg_modules_per_contigs.tsv.gz
+│   │   ├── ERZ12345_kegg_modules_per_contigs.tsv.gz.gzi
+│   │   ├── ERZ12345_kegg_modules_summary.tsv.gz
+│   │   └── ERZ12345_kegg_modules_summary.tsv.gz.gzi
+│   └── dram-distill
+│       ├── ERZ12345_dram.tsv.gz
+│       ├── ERZ12345_dram.html.gz
+│       ├── ERZ12345_genome_stats.tsv.gz
+│       └── ERZ12345_metabolism_summary.xlsx.gz
+└── gff
+```
+
+#### Output files - antismash
+
+This subdirectory contains the outputs of running `AntiSMASH` on the proteins of the assembly, describing the detected BGCs and outputting them in multiple different formats.
+
+- **ERZ12345_antismash.gbk.gz**: This `gbk` file contains the different AntiSMASH annotations in the GenBank format.
+- **ERZ12345_antismash.gff.gz**: This `gff` file contains the different AntiSMASH annotations in the GFF3 format.
+- **ERZ12345_merged.json**: This `json` file contains the different AntiSMASH annotations in a JSON object file.
+- **ERZ12345_antismash_summary.tsv.gz**: This `tsv` file contains the different AntiSMASH labels in the proteins of the assembly.
+
+#### Output files - sanntis
+
+This subdirectory contains the outputs of running `SanntiS` on the proteins of the assembly, also describing the detected BGCs using this new machine learning-based tool.
+
+- **ERZ12345_sanntis.gff.gz**: This `gff` file contains the different SanntiS annotations in the GFF3 format.
```

Comment:
Not for this PR probably, but having the GFF (mostly 9th column) spec is useful. For SanntiS we can just link to the repo.

```suggestion
- **ERZ12345_sanntis.gff.gz**: This `gff` file contains the different SanntiS annotations in the GFF3 format. The GFF3 specification can be found in the [tool repo](https://github.com/Finn-Lab/SanntiS?tab=readme-ov-file#ouput) 
```